### PR TITLE
[integration-k8s-packet#105] SR-IOV config

### PR DESCRIPTION
**Pull request by Bolodya1997**
## Description
Adds documentation about SR-IOV configuration.

## Issue link
Closes networkservicemesh/integration-k8s-packet#105.

## How Has This Been Tested?

- [ ] Added unit testing to cover
- [ ] Tested manually
- [ ] Tested by integration testing
- [ ] Have not tested

## Types of changes

- [ ] Bug fix
- [ ] New functionallity
- [x] Documentation
- [ ] Refactoring
- [ ] CI

### CODE REVIEW

File: examples/ovs/README.md [hunk 1]
```diff
@@ -11,6 +11,26 @@ Contain basic setup for NSM that includes `nsmgr`, `forwarder-ovs`, `registry-k8
 - [Kernel to Kernel Connection](../use-cases/Kernel2Kernel)
 - [SmartVF to SmartVF Connection](../use-cases/SmartVF2SmartVF)
 
+## SR-IOV config
```
Comment (pperiyasamy) [inline examples/ovs/README.md:14]: @Bolodya1997 The SR-IOV config is bit different for SmartNIC case, Can you just follow this [documentation](https://github.com/k8snetworkplumbingwg/ovs-cni/blob/main/docs/ovs-offload.md#mellanox-sr-iov-configuation) (except OVS bridge configuration) ?

Comment (Bolodya1997) [reply examples/ovs/README.md:14]: This PR is about **what should be SR-IOV config file** not about **how NICs should be configured**, so I guess such additional information can be added with some another PR if you want :)

**Review by pperiyasamy — APPROVED** [review]

File: examples/ovs/README.md (hunk 1 above)
Comment (pperiyasamy) [reply examples/ovs/README.md:14]: @Bolodya1997 Sure, I get it.

**Review by denis-tingaikin — APPROVED** [review]

**Comment by denis-tingaikin** [comment]
@edwarnicke Be free to merge it if you have no comments

**Pull request merged**